bin/kano-empty-trash

```diff
@@ -42,17 +43,44 @@ def empty_all_trash():
         # it thinks it must be root if sudo is in effect, but this is not the case here.
         os.system('sudo -H -u#{} SUDO_USER={} /usr/bin/kano-empty-trash '.format(uid, uname))
 
+def clean_old_dumps(dir, age):
+    '''
+    Cleans the core dump files which are set by the kernel.core_pattern
+
+    Args:
+        dir (str): This is where core dump files live.
+        age (int): List files older than age days old.
+    Raises:
+        OSError: Raised if the core dump dir isn there
 
+    TODO:
+        * Get the coredump_dir directly from an environment variable or through
+        the configuration file of kernel.core_pattern 
+    '''
+    if dir:
```

Comment:
I'd swap this to be

```python
if not dir:
    raise AttributeError
```

to simplify the code and make it obvious that this is checking input arguments and all code after is the code proper.

Also, shouldn't there be a line in the comment above that it can also raise `AttributeError`?